packages/client/lib/miner/miner.ts

```diff
@@ -97,8 +143,8 @@ export class Miner {
     }
     this.running = true
     this.config.events.on(Event.CHAIN_UPDATED, this.chainUpdated.bind(this))
-    void this.queueNextAssembly() // void operator satisfies eslint rule for no-floating-promises
     this.config.logger.info(`Miner started. Assembling next block in ${this.period / 1000}s`)
```

Comment:
This is cosmetic but this should say something different if the consensus algorithm is PoW.  Maybe something like.  You'll have to adjust the formatting because Prettier and ESLint can't agree..
```suggestion
this.config.logger.info(
      `Miner started. ${this.config.execCommon.consensusType() === ConsensusType.ProofOfAuthority
        ? `Assembling next block in ${this.period / 1000}s`
        : `Finding next PoW solution 🔨`
      }`
    )
```